packages/editor/src/components/ExamplesBrowser.tsx

```diff
@@ -9,25 +9,44 @@ export default function ExamplesBrowser() {
   if (examples.state !== "hasValue") {
     return <span>loading examples...</span>;
   }
+
+  const parser = new DOMParser();
+  const serializer = new XMLSerializer();
+
   return (
     <div>
-      {examples.contents.map((example, k) => (
-        <div key={example.name}>
-          <FileButton
-            key={k}
-            isFocused={false}
-            onClick={() => loadExample(example)}
-          >
-            <div style={{ display: "flex", justifyContent: "space-between" }}>
-              {/* <div
-                style={{ width: 50, height: 50 }}
-                dangerouslySetInnerHTML={{ __html: example.preview! }}
-              ></div> */}
-              {example.name}
-            </div>
-          </FileButton>
-        </div>
-      ))}
+      {examples.contents.map((example, k) => {
+        const svgDoc = parser.parseFromString(
+          example.preview!,
+          "image/svg+xml"
+        );
+        const cropped = svgDoc.querySelector("croppedViewBox")?.innerHTML;
```

Comment:
```suggestion
        const cropped = svgDoc.querySelector("croppedViewBox");
```

See below for doing stuff after `undefined` check